hpolib/abstract_benchmark.py

```diff
@@ -114,7 +120,23 @@ def wrapper(self, configuration: Union[np.ndarray, ConfigSpace.Configuration, Di
                                 f'was {type(configuration)}')
 
             self.configuration_space.check_configuration(config)
-            return foo(self, configuration, **kwargs)
+
+            # Check the fidelities, if any
+            fidelity = self.opt_fidelity.get_dictionary()
```

Comment:
Shouldn't this get get_fidelity_space?

Reply:
Why should it? This might be wrapped up with the one below.

Reply:
Is this still relevant? The comment is outdated?

Reply:
Ah I missed that. Still new to the review interface. I'll resolve this conversation.